### PR TITLE
fixing invalid relative paths in pom.xml

### PR DESCRIPTION
**Pull request by rasika**

### CODE REVIEW

File: modules/samples/virtual_firealarm/src/org.wso2.carbon.device.mgt.iot.sample.virtual.firealarm.service.impl/src/main/webapp/META-INF/resources.xml [hunk 1]
```diff
@@ -50,14 +50,14 @@
     <Resource>
         <AuthType>Any</AuthType>
         <HttpVerb>GET</HttpVerb>
-        <Uri>http://localhost:9763/virtual_firealarm/manager/device</Uri>
-        <UriTemplate>/manager/device/*</UriTemplate>
+        <Uri>http://localhost:9763/virtual_firealarm/manager/device/{device_id}</Uri>
```
Comment (DilanUA) [inline modules/samples/virtual_firealarm/src/org.wso2.carbon.device.mgt.iot.sample.virtual.firealarm.service.impl/src/main/webapp/META-INF/resources.xml:53]: why do we need to place localhost and 9763 port here? and this does not support https either. Cannot we simply keep the relative path here?

File: modules/samples/virtual_firealarm/src/org.wso2.carbon.device.mgt.iot.sample.virtual.firealarm.service.impl/src/main/webapp/META-INF/resources.xml [hunk 2]
```diff
@@ -50,14 +50,14 @@
...
+        <UriTemplate>/manager/device/{device_id}</UriTemplate>
     </Resource>
     <Resource>
         <AuthType>Any</AuthType>
         <HttpVerb>GET</HttpVerb>
-        <Uri>http://localhost:9763/virtual_firealarm/manager/devices</Uri>
-        <UriTemplate>/manager/devices/*</UriTemplate>
+        <Uri>http://localhost:9763/virtual_firealarm/manager/devices/{username}</Uri>
```
Comment (DilanUA) [inline modules/samples/virtual_firealarm/src/org.wso2.carbon.device.mgt.iot.sample.virtual.firealarm.service.impl/src/main/webapp/META-INF/resources.xml:59]: why do we need to place localhost and 9763 port here? and this does not support https either. Cannot we simply keep the relative path here?

File: modules/samples/virtual_firealarm/src/org.wso2.carbon.device.mgt.iot.sample.virtual.firealarm.service.impl/src/main/webapp/META-INF/resources.xml (hunk 1 above)
Comment (rasika) [inline modules/samples/virtual_firealarm/src/org.wso2.carbon.device.mgt.iot.sample.virtual.firealarm.service.impl/src/main/webapp/META-INF/resources.xml:53]: This was added to publish api details into api-publisher when the web-app started. Hence relative-path does not work for api-publisher. This was later replaced to pick up api-details with swagger API annotations.

File: modules/samples/virtual_firealarm/src/org.wso2.carbon.device.mgt.iot.sample.virtual.firealarm.service.impl/src/main/webapp/META-INF/resources.xml (hunk 2 above)
Comment (rasika) [inline modules/samples/virtual_firealarm/src/org.wso2.carbon.device.mgt.iot.sample.virtual.firealarm.service.impl/src/main/webapp/META-INF/resources.xml:59]: This was added to publish api details into api-publisher when the web-app started. Hence relative-path does not work for api-publisher. This was later replaced to pick up api-details with swagger API annotations.

**Pull request merged**